Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -146,6 +162,7 @@ def livecheck_formulae(formulae_to_check, args)
         Homebrew.failed = true
 
         if args.json?
+          progress.increment
```

Comment:
```suggestion
          progress.increment if progress
```

I missed this one the first time around.